ingest/expression_files/expression_files.py

```diff
@@ -205,7 +205,13 @@ def get_study_expression_file_ids(
         is_raw_count_files = GeneExpression.is_raw_count_file(
             study_id, current_study_file_id, client
         )
-        if is_raw_count_files:
+        if not is_raw_count_files:
+            # Legacy data entries will not have the embeded document 'expression_file_info'
+            QUERY["$or"] = [
```

Comment:
Are we 100% certain that this `$or` is still scoped by the rest of the query?  Meaning the `$and` and `$nor` are still honored even if the `$or` is present?